src/igvfd/schemas/donor.json

```diff
@@ -83,6 +83,7 @@
         },
         "external_resources": {
             "title": "External Resources",
+            "comment": "External identifier that uniquely identify the donor of the sample, e.g. for strains it might be a strain identifier at a repository or a de-identified donor ID for a person.",
             "description": "A list of external resources associated with the donor.",
             "type": "array",
             "minItems": 1,
```

Comment:
(Regarding below, line 90) This array has "uniqueItems" = "false". I am trying to think of a case where we would want duplicates of external_resources, but can't think of any yet. Should we change it to "true"?

Reply:
agree should be true